evap/evaluation/tools.py

```diff
@@ -141,7 +144,7 @@ def calculate_results(course):
     if course.state != "published":
         return _calculate_results_impl(course)
 
-    cache_key = str.format('evap.staff.results.tools.calculate_results-{:d}', course.id)
+    cache_key = 'evap.staff.results.tools.calculate_results-%d' % course.id
```

Comment:
please use .format, see the note here: https://docs.python.org/3.5/library/stdtypes.html#printf-style-string-formatting